src/java/us/kbase/mobu/tester/ConfigLoader.java

```diff
@@ -47,7 +47,12 @@ public ConfigLoader(Properties props, boolean testMode,
         String tokenString = props.getProperty(testPrefix + "token");
         if (tokenString != null && tokenString.trim().isEmpty()) {
             tokenString = null;
+            String test_token = System.getenv("KBASE_TEST_TOKEN");
+            if (test_token != null && !test_token.trim().isEmpty()) {
+                tokenString = test_token;
+            }
```

Comment:
Default to using the value in `test.cfg`, and check the env var `KBASE_TEST_TOKEN` if that value is not set.